packages/data/src/world/oot/overworld.yml

```diff
@@ -2371,7 +2387,7 @@
     BUGS: "is_child && has_bottle"
   locations:
     "Gerudo Valley Crate HP": "is_child || can_longshot || climb_anywhere"
-    "Gerudo Valley Waterfall HP": "true"
+    "Gerudo Valley Waterfall HP": "is_child || can_swim || can_longshot || (has_iron_boots && can_hookshot)" ##Grab the PoH with Cucco as child. The Hookshot can attach to the ladder.
```

Comment:
Longshot from where though? The ledge where the gossip stone is?

Reply:
Yes, you can make it from there.